azure/functions_worker/bindings/datumdef.py

```diff
@@ -0,0 +1,82 @@
+from .. import protos
+
```

Comment:
Nit: Can you rename this class to rpcConverters as this is a utility to convert To and From TypedData?

Reply:
Not really.  The `Datum` class represents a piece of data.  It can convert to and from `TypedData`, but that's not the only way it gets used.